lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -16,6 +16,7 @@ defmodule ExUnit.CLIFormatter do
       colors: colors(opts),
       width: get_terminal_width(),
       slowest: opts[:slowest],
+      slowest_modules: opts[:slowest_modules],
```

Comment:
was name :slowest_cases considered? I thought case is a more common used name for this even though it is of cluese modules under the hood.

Reply:
you have a point, i didn't think of it. My only thought about this is that a single test is also a case, isn't it?

Reply:
Yes, we moved away from the case naming because it was ambiguous. Today we have Test and TestModule.